src/amber/environment.cr

```diff
@@ -0,0 +1,40 @@
+require "./environment/**"
+require "./support/file_encryptor"
+
+module Amber::Environment
+  alias EnvType = String | Symbol
+
+  macro included
+    SECRET_KEY  = "AMBER_ENCRYPTION_KEY"
+    SECRET_FILE = "./.encryption_key"
+    CURRENT_ENVIRONMENT = ENV["AMBER_ENV"] ||= "development"
+    class_property environment_path : String = "./config/environments/"
+
+    @@_settings : Settings?
+
+    def self.settings
+      @@_settings ||= EnvironmentLoader.new(CURRENT_ENVIRONMENT, @@environment_path, file_encryptor).settings
```

Comment:
WE need to be aware that loading settings here will cause the entire CLI to exception if ENV is encrypted and they don't have the key. This could lead to confusion. IF `AMBER_ENV=production` and someone calls `amber g scaffold` or `amber routes` or any call not related to the environment it will still error.

Before this PR only a call to `amber database` or `amber encrypt` would have caused a CLI error.

Reply:
The thought behind this is that we can have access to the same functionality from the cli to work with the environment.

Reply:
Yeah I totally get that. I'm just afraid that it will lead to issues. We could solve them by catching the exception in this case and loading defaults.